src/app/shared/ErrorHandler/ErrorHandler.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Spinner from '../Spinner';
+import ErrorHandler from './ErrorHandler';
+
+describe('<ErrorHandler />', () => {
+  it('should match its snapshot', () => {
+    const wrapper = shallow(<ErrorHandler />);
+
+    expect(wrapper).toMatchSnapshot();
+  });
+});
+
+describe('When no JS errors are caught in a child component', () => {
```

Comment:
Why do you test if <Spinner /> renders? This soupose to be an independent component.